frontend/components/AddSite/index.js

```diff
@@ -35,21 +38,12 @@ class AddSite extends React.Component {
     this.onSubmit = this.onSubmit.bind(this);
   }
 
-  defaultOwner() {
-    const userState = this.props.storeState.user
-    if (userState.data) {
-      return userState.data.username
-    } else {
-      return ""
-    }
-  }
-
   onSubmit(event) {
     event.preventDefault();
     siteActions.addSite(this.state);
   }
 
-  onSubmitTemplate(site) {
+  onSubmitTemplate(site) { // eslint-disable-line class-methods-use-this
```

Comment:
Why did you ignore the rule here? I ran into it in a different component, it'd be a nice one to disable globally.